### DISCUSSION TITLE
Find the odd one

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#932] [Galaxy Trivia] [answered]
**IRAMVEP**
**LAREWOWF**
**LOKTEENS**
**WEELPASRE**

**Comment by tomerf** [comment]
IRAMVEP

**Comment by peterbaccaro** [comment]
1. VAMPIRE
2. WAREWOLF
3. SKELETON
4. SLEEPWEAR

The odd one out is **WEELPASRE (i.e. SLEEPWEAR)** because it does not belong to the supernatural or mythical creature category like the other three options (VAMPIRE, WAREWOLF, SKELETON).

**Comment by YashNuhash** [comment, marked as answer]
> IRAMVEP        -> VAMPIRE
> LAREWOWF   -> WAREWOLF
> LOKTEENS      -> SKELETON
> WEELPASRE    -> SLEEPWEAR

The odd one out is SLEEPWEAR.

The other three words, VAMPIRE, WAREWOLF, and SKELETON are all associated with the supernatural or horror genres. SLEEPWEAR, on the other hand, is simply a type of clothing.
